aemedge/blocks/channel-lookup/channel-lookup.css

```diff
@@ -75,13 +75,10 @@
     font-weight: 700;
 }
 
-.section .default-content-wrapper {
-    max-width: 920px;
-    margin: 0 auto;
-}
-
-.section .default-content-wrapper  p {
+.section .default-content-wrapper  h4 {
```

Comment:
Can you change the heading on this to be a bold P tag, or an H3? (use the same style though). This is to comply with the accessibility score requirements. There's an H1 and H2 in the hero area, so it doesn't like when H4 is next (skipping the H3).
![image](https://github.com/user-attachments/assets/c1b5a0c7-d28c-488f-be91-dbf99e2571fb)